.gitlab-ci.yml

```diff
@@ -1,34 +1,83 @@
-.bazel_docker_image: &bazel_docker_image thulioassis/bazel-docker-image:0.29.1
-.default_branch: &default_branch master
+image: thulioassis/bazel-docker-image:0.29.1
```

Comment:
I changed to where the images were published, but I couldn't get to use the github ones without logging in. I may move it back to dockerhub.

Reply:
Oh, publishing it on docker hub is fine for me; I just want a link to where the `Dockerfile` (and associated files) live, so that it can get rebuilt if needed.